lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -769,21 +781,24 @@ ConstExprFunctionState::computeOpaqueCallResult(ApplyInst *apply,
   if (callee->hasSemanticsAttr("arc.programtermination_point")) {
     // TODO: Actually get the message out of fatalError.  We can literally get
     // its string and file/line/col info and propagate it up!
-    return SymbolicValue::getUnknown((SILInstruction *)apply,
-                                     UnknownReason::Trap);
+    return SymbolicValue::getUnknown(
```

Comment:
I'd suggest adding a "getUnknown" method on the enclosing class that takes a node + reason, but handle passing the callStack and allocator.